src/com/epam/javacourse/homework4/GreenhouseManagement.java

```diff
@@ -1,14 +1,20 @@
 package com.epam.javacourse.homework4;
 
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Scanner;
 
 public class GreenhouseManagement implements Greenhouse {
+    private static final Path FILE_NAME = Paths.get("C:\\Users\\Sviatlana_Shedava\\IdeaProjects\\SviatlanaShedava5\\src\\com\\epam\\javacourse\\homework4\\plants.txt");
+    private static final String DELIMITER = ";";
```

Comment:
This variable is used only once and in only one method. And it's not enough to make it a global class variable.